week1/assignment/가계부 💸/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="./img/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>수빈 | 가계부</title>
+</head>
+
+<body>
+    <header>
+        <h1>수빈이의 가계부</h1>
+    </header>
+    <section id="asset">
+        <h2 class="subtitle">나의 자산</h2>
+        <b id="money">3,432,000</b>
+        <div id="today">
+            <img src="./img/triangle-up.png" alt="위쪽화살표" />
+            <p class="income">150000</p>
+            <img src="./img/triangle-down.png" alt="아래쪽화살표" />
+            <p class="spending">43200</p>
+        </div>
+    </section>
+    <section id="main">
+        <h2 class="subtitle">내역 리스트</h2>
+        <div id="date">
+            <button id="prev-day">◀</button>
+            <p>10월 9일</p>
+            <button id="next-day">▶</button>
+        </div>
+        <div id="money-type">
+            <input type="checkbox" checked="on" id="type-income" />
+            <label for="type-income">수입</label>
+            <input type="checkbox" checked="on" id="type-spending" />
```

Comment:
checked 속성은 boolean값이라서 속성을 주는 것 자체가 true를 의미해요! 그래서 요렇게만 쓰시면 된답니당!
```suggestion
<input type="checkbox" checked id="type-income" />
            <label for="type-income">수입</label>
            <input type="checkbox" checked id="type-spending" />
```